velox/experimental/wave/common/tests/CMakeLists.txt

```diff
@@ -28,9 +28,14 @@ add_test(velox_wave_common_test velox_wave_common_test)
 set_tests_properties(velox_wave_common_test PROPERTIES LABELS cuda_driver)
 
 target_include_directories(velox_wave_common_test PRIVATE ../../../breeze)
+target_compile_definitions(
+  velox_wave_common_test
+  PRIVATE
+    PLATFORM_CUDA
+    CUDA_PLATFORM_SPECIALIZATION_HEADER=${CUDA_PLATFORM_SPECIALIZATION_HEADER})
 if(VELOX_SKIP_WAVE_BRANCH_KERNEL_TEST)
-  target_compile_definitions(velox_wave_common_test
-                             PRIVATE VELOX_SKIP_WAVE_BRANCH_KERNEL_TEST=1)
+  add_compile_definitions(velox_wave_common_test PRIVATE
```

Comment:
`target_compile_definitions` is correct here given the use of a target and a visibility scope.

Reply:
Yes, this has been restored in latest version of PR where I'm introducing a breeze_cuda interface library, which makes it easy to depend on breeze inside wave.